internal/versionlock/versionlock.go

```diff
@@ -0,0 +1,25 @@
+// Package versionlock defines structures for creating a dnf versionlock
+// configuration file as described in dnf-versionlock(8) and
```

Comment:
This is not true for `dnf-versionlock(8)`. On Fedora with DNF5, this will point to the DNF5 version of versionlock. However, the DNF4 version has a different format; it is not TOML